### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ryang1428**
Now presenting in a window
- Added new Modal examples to the sample project

### CODE REVIEW

File: Pod/Classes/URBNAlertController.m [hunk 1]
```diff
@@ -28,7 +30,7 @@ + (instancetype)sharedInstance {
     dispatch_once(&onceToken, ^{
         instance = [[URBNAlertController alloc] init];
         [instance setAlertStyler:[URBNAlertStyle new]];
-        instance.window = [[UIApplication sharedApplication] windows][0];
+        instance.presentingWindow = [[UIApplication sharedApplication] windows][0];
```
Comment (KVTaniguchi) [inline Pod/Classes/URBNAlertController.m]: is using .firstObject any better / worse than [0]? if its nil won't [0] crash?

File: Pod/Classes/URBNAlertController.m [hunk 2]
```diff
@@ -102,6 +117,20 @@ - (void)dismissAlert {
 }
 
 #pragma mark - Methods
+- (void)setupAlertWindow {
+    if (self.alertWindow) {
+        return;
+    }
+    
+    self.presentingWindow = [[UIApplication sharedApplication] windows][0];
```
Comment (KVTaniguchi) [inline Pod/Classes/URBNAlertController.m]: same question as above

File: Pod/Classes/URBNAlertController.m (hunk 1 above)
Comment (jgrandelli) [inline Pod/Classes/URBNAlertController.m]: :+1:

File: Pod/Classes/URBNAlertController.m (hunk 2 above)
Comment (jgrandelli) [inline Pod/Classes/URBNAlertController.m]: :+1:

**Pull request merged**